rosdep/base.yaml

```diff
@@ -2182,6 +2182,17 @@ lib32asound2:
   fedora: ['alsa-lib(%{__isa_name}-32)', alsa-lib]
   gentoo: [media-libs/alsa-lib]
   ubuntu: [lib32asound2]
+libabsl-dev:
+  arch: [abseil-cpp]
+  debian:
+    '*': [libabsl-dev]
+    buster: null
+  fedora: [abseil-cpp-devel]
+  gentoo: [dev-cpp/abseil-cpp]
+  ubuntu:
```

Comment:
```suggestion
  rhel:
    '*': [abseil-cpp-devel]
    '7': null
  ubuntu:
```

Reply:
Ah, I forgot to look at the comment section.  Thanks, applied now.